server/services/post-secondary-institutes.js

```diff
@@ -0,0 +1,32 @@
+const { dbClient, collections } = require('../db');
+
+const getPSIs = async () => dbClient.db[collections.POST_SECONDARY_INSTITUTIONS].find();
+
+const getPSI = async (id) =>
+  dbClient.db[collections.POST_SECONDARY_INSTITUTIONS].find({
+    id,
+  });
+
+const makePSI = async (psi) => {
+  const data = {
+    institute_name: psi.instituteName,
+    health_authority: psi.healthAuthority,
+    postal_code: psi.postalCode,
+  };
+
+  try {
+    const newPSI = await dbClient.db[collections.POST_SECONDARY_INSTITUTIONS].insert(data);
+    return newPSI;
+  } catch (error) {
+    if (error.code === '23505') {
+      return { error: 'Duplicate Name for PSI', status: '23505' };
```

Comment:
```suggestion
      return { error: 'Duplicate Name for PSI', code: '23505' };
```

I think `status` should be reserved for HTTP status' in this context, can this stay as `code`?